source/reference/minio-mc-admin/mc-admin-accesskey-create.rst

```diff
@@ -0,0 +1,232 @@
+
+   .. code-block::
+
+      --expiry-duration 30d
+
+   Mutually exclusive with :mc-cmd:`~mc admin accesskey create --expiry`.
+
+.. mc-cmd:: --name
+   :optional:
+
+   Add a human-readable name for the service account.
+
+.. mc-cmd:: --policy
+   :optional:
+
+   The readable path to a :ref:`policy document <minio-policy>` to attach to the new access key, with a maximum size of 2048 characters.
+   The attached policy cannot grant access to any action or resource not explicitly allowed by the parent user's policy or group policies
+
+.. mc-cmd:: --secret-key
+   :optional:
+
+   The secret key to associate with the new account.
+   Omit to let MinIO autogenerate a random 40-character value.
+
+
+Global Flags
+~~~~~~~~~~~~
+
+.. include:: /includes/common-minio-mc.rst
+   :start-after: start-minio-mc-globals
+   :end-before: end-minio-mc-globals
+
+
+Examples
+--------
+
+Create access key / secret key pair for the authenticated user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command generates a new and random access key and random secret key pair for the user currently logged in to MinIO deployment at the alias ``myminio``.
+The access key and secret key have the same access policies as the authenticated user.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin accesskey create myminio/
+
+Create a custom access key / secret key pair for the authenticated user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command creates a new access key and secret key pair for the user currently logged in to MinIO at the alias ``myminio``.
+The access key and secret key have the same access policies as the authenticated user.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin accesskey create myminio/ --access-key myaccesskey --secret-key mysecretkey 
+
+Create an access key / secret key pair for another user with limited duration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command creates a new access key and secret key pair for a user, ``miniouser`` on the alias ``myminio``.
+The access key and secret key have the same access policies as ``miniouser``.
+The credentials remain valid for 24 hours after creation.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin accesskey create myminio/ miniouser --expiry-duration 24h
+
+
+Create access key / secret key pair for the authenticated user that expires
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following command generates a new and random access key and random secret key pair for the user currently logged in to MinIO deployment at the alias ``myminio``.
+The access key and secret key have the same access policies as the authenticated user.
+The credentials expire on the fifteenth day of January, 2025.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin accesskey create myminio/ --expiry 2025-01-15
```

Comment:
Is this the required date style? If so, we _could_ describe it as ISO 8601. Or less formally "something something year month day"

Reply:
All of the allowed date styles are already listed under the `--expire` flag.

Reply:
Added a link to the flag.